internal/tools/pipeline/providers/edgepipeline_register/edge_client.go

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package edgepipeline_register
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/mohae/deepcopy"
+
+	"github.com/erda-project/erda/apistructs"
+	"github.com/erda-project/erda/pkg/discover"
+	"github.com/erda-project/erda/pkg/http/httpserver"
+	"github.com/erda-project/erda/pkg/strutil"
+)
+
+var (
+	ClientsHookApiPath = "/api/pipeline-clients/actions/hook"
```

Comment:
what is pipeline client

Reply:
adjusted, EdgeHookApiPath = "/api/pipeline-edge/actions/hook"